.vscode/cesiumjs.code-snippets

```diff
@@ -0,0 +1,44 @@
+{
+	// Place your workspace snippets here. Each snippet is defined under a snippet name and has a scope, prefix, body and
+	// description. Add comma separated ids of the languages where the snippet is applicable in the scope field. If scope
+	// is left empty or omitted, the snippet gets applied to all languages. The prefix is what is
+	// used to trigger the snippet and the body will be expanded and inserted. Possible variables are:
+	// $1, $2 for tab stops, $0 for the final cursor position, and ${1:label}, ${2:another} for placeholders.
+	// Placeholders with the same ids are connected.
+	// Example:
+	// "Print to console": {
+	// 	"scope": "javascript,typescript",
+	// 	"prefix": "log",
+	// 	"body": [
+	// 		"console.log('$1');",
+	// 		"$2"
+	// 	],
+	// 	"description": "Log output to console"
+	// }
+	"Debug Pragma": {
+		"scope": "javascript",
+		"prefix": "pragdebug",
+		"description": "Insert or wrap selection with debug pragma comments",
+		"body": [
+			"//>>includeStart('debug', pragmas.debug);",
+			"${TM_SELECTED_TEXT}$0",
```

Comment:
This also lets you quickly wrap already existing code if you forgot the pragma.


https://github.com/user-attachments/assets/1d0a9786-204f-4dd9-94c6-a6a00c5f73dd